running-your-first-ot-server/setup-webserver-linux.md

```diff
@@ -0,0 +1,178 @@
+# Setting up your ubuntu linux server
+guiding to setup your very own ubuntu linux server with a webserver plus the ability to deploy your tfs server
+
+> We are not responsible for the mistakes you make
+
+# Getting started
+Rent a ubuntu server 18.04 or higher from a trusted hosting company such as ovh.com or install for the sake of learning experience ubuntu server 18.04 or higher on another computer or if you are on windows use virtualbox to get your ubuntu server.
```

Comment:
Avoid naming specific companies in the official docs (remove ovh.com)

Reply:
Proper capitalization needed (`Ubuntu Server` not `ubuntu server`), also the operating system isn't really tied to the physical server you rent.